runner/config.ts

```diff
@@ -17,6 +17,7 @@ import * as fs from 'fs';
 import * as _ from 'lodash';
 import * as nomnom from 'nomnom';
 import * as path from 'path';
```

Comment:
something to keep in mind: path's behavior changes based on the machine it's running on (ex: outputs/expects `/` for Unix & `\` for windows). I see you using it in a few places below, consider using `path.posix` if you want to force URL-compliant slashes on all machines.

https://nodejs.org/api/path.html#path_path_posix
ex: `path.posix.relative(...`

Reply:
great catch